Routing.md

```diff
@@ -0,0 +1,59 @@
+# Routing Meatdata Payload Extension
+
+## Introduction
+Specify a metadata payload for [RSocket](https://rsocket.io) that contains both information about where to route a [`PAYLOAD` frame](https://github.com/rsocket/rsocket/blob/master/Protocol.md#frame-payload) as well as metadata for the frame itself.
+
+## Metadata Payload
+The metadata payload has a MIME Type of `message/x.rsocket.routing`.
+
+### Payload Contents
+```
+     0                   1                   2                   3
+     0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
+    +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    |            Version            |T|P|S|D|   Flags   |
+    +-------------------------------+-+-+-+-+-----------+-----------+
+    |      Route Token Length       |   Route Token                ...
+    +-------------------------------+-------------------------------+
+    |                                                               |
+    +                                                               +
+    |                                                               |
+    +                           Trace ID                            +
+    |                                                               |
+    +                                                               +
+    |                                                               |
+    +---------------------------------------------------------------+
+    |                                                               |
+    +                           Span ID                             +
+    |                                                               |
+    +---------------------------------------------------------------+
+    |                                                               |
+    +                        Parent Span ID                         +
+    |                                                               |
+    +-+-+-----------------------------------------------------------+
+    |S|D|
```

Comment:
Why are these two bits not encoded in the flags? I assume I’m missing something.

Reply:
Splitting out tracing into a separate extension spec.